test/tests/postprocessors/pseudotimestep/gold/steady_state_out.csv

```diff
@@ -0,0 +1,2 @@
+time,pseudotimestep
+50,20.637741707231
```

Comment:
The final timestep size didn't change very much, is this an analytic result that can be easily verified for each type?  If not, is there someway to setup a test object that would show this timestepper is working as expected?

Reply:
@lynnmunday still not sure what you thought is missing here. I updated the test cases and there is a close to an  analytic result I set it up for, but I still check via CSVDiff.